### DISCUSSION TITLE
Shrinking down everything selected

### DISCUSSION THREAD

**Discussion by KlaraHirsch** [jgraph/drawio#3683] [Q&A] [answered]
Hi, I am new to draw.io!

I have a simple question, how can I shrink down everything that I have selected? If I try it, I only shrink down one rectangle.

<img width="740" alt="image" src="https://github.com/jgraph/drawio/assets/137304816/c620331a-0d27-4457-a31c-12f51264b554">

Thanks in advance!

**Comment by egrimate** [comment, marked as answer]
Make a group of the selection, resize the group and then ungroup the selection.

**Reply by KlaraHirsch** [reply]
Thanks egrimate!
